controllers/vulnerabilityreport/vulnerabilityreport_controller.go

```diff
@@ -194,8 +194,9 @@ func getCountPerSeverity(report *aqua.VulnerabilityReport) map[string]float64 {
 		string(aqua.SeverityHigh):     float64(report.Report.Summary.HighCount),
 		string(aqua.SeverityMedium):   float64(report.Report.Summary.MediumCount),
 		string(aqua.SeverityLow):      float64(report.Report.Summary.LowCount),
-		string(aqua.SeverityNone):     float64(report.Report.Summary.NoneCount),
-		string(aqua.SeverityUnknown):  float64(report.Report.Summary.UnknownCount),
+		// aqua.SeverityNone was in starboard but removed in trivy-operator
+		// string(aqua.SeverityNone):    float64(report.Report.Summary.NoneCount),
```

Comment:
Maybe we can just remove it then. We'll just have to test this with existing VulnerabilityReports and see if they can still be reconciled.